test/Preprocessing/lit.local.cfg.py

```diff
@@ -0,0 +1,12 @@
+# -*- Python -*-
+
+import os
+import platform
+import sys
+
+import lit.formats
+from lit.llvm import llvm_config
+
+# Added this lit config file to prevent lit from discovering these tests
```

Comment:
Pedantry, but the presence of the file itself does not do this. It is the assignment to config.suffixes that does it. Change the comment to explain that.